apisix/stream/xrpc/sdk.lua

```diff
@@ -108,6 +108,10 @@ function _M.get_req_ctx(session, id)
     local ctx = core.tablepool.fetch("xrpc_ctxs", 4, 4)
     -- fields start with '_' should not be accessed by the protocol implementation
     ctx._id = id
+    core.ctx.set_vars_meta(ctx)
+    ctx.conf_id = tostring(session._route.protocol)
```

Comment:
Better to set a per-logger conf_id because multiple instances of the same  logger plugin can be used together.

Reply:
update